pypesto/storage/hdf5.py

```diff
@@ -0,0 +1,57 @@
+"""Convenience functions for working with HDF5 files"""
+import h5py
+from typing import Collection
+from numbers import Number
+
+
+def write_string_array(f: h5py.Group,
+                       path: str,
+                       strings: Collection) -> None:
+    """
+    Write string array to hdf5
+
+    Arguments:
+        f: h5py.File
+        path: path of the dataset to create
+        strings: list of strings
```

Comment:
```suggestion
    Parameters
    ----------
        f: h5py.File
        path: path of the dataset to create
        strings: MORE SPECIFIC?
```
reason for the "Parameters" (please edit in other places too): We use the numpy docstring format. Not sure if something will break if using this format here.